File: python/src/exams/exams2/Exam103.py
""" 103-与车相交的点

"""

from typing import List


def numberOfPoints(nums: List[List[int]]) -> int:
    c = 0

    for i in range(1, 101):
        for pair in nums:
            if pair[0] <= i <= pair[1]:
                c += 1
                break

    return c

File: kotlin/src/main/kotlin/exams/exams2/Exam103.kt
package exams.exams2

fun numberOfPoints(nums: List<List<Int>>): Int {
    var c = 0

    for (i in 1..100) {
        for (pair in nums)
            if (i in pair[0]..pair[1]) {
                c++
                break
            }
    }

    return c
}
